storage_backend_s3/views/backend_storage_view.xml

```diff
@@ -6,12 +6,20 @@
     <field name="arch" type="xml">
         <group name="config" position="after">
             <group name="Amazon S3" string="Amazon S3" attrs="{'invisible': [('backend_type', '!=', 'amazon_s3')]}">
-                <field name="aws_host" />
-                <field name="aws_access_key_id" attrs="{'required': [('backend_type', '=', 'amazon_s3')]}"/>
-                <field name="aws_secret_access_key" attrs="{'required': [('backend_type', '=', 'amazon_s3')], 'readonly':[('backend_type', '!=', 'amazon_s3')]}"/>
-                <field name="aws_region" attrs="{'required': [('backend_type', '=', 'amazon_s3'),('aws_host','=',False)]}"/>
-                <field name="aws_bucket"  attrs="{'required': [('backend_type', '=', 'amazon_s3')]}"/>
-                <field name="aws_cache_control"/>
+                <group>
+                    <field name="aws_host" />
+                    <field name="aws_access_key_id" attrs="{'required': [('backend_type', '=', 'amazon_s3')]}"/>
+                    <field name="aws_secret_access_key" attrs="{'required': [('backend_type', '=', 'amazon_s3')], 'readonly':[('backend_type', '!=', 'amazon_s3')]}"/>
+                    <field name="aws_bucket"  attrs="{'required': [('backend_type', '=', 'amazon_s3')]}"/>
+                    <field name="aws_cache_control"/>
+                </group>
+                <group>
+                        <field name="aws_region" widget="radio"
```

Comment:
@mileo IMO we should use the default selection witdget for  `aws_region` (not blocking at all)
![image](https://user-images.githubusercontent.com/544090/68648789-89ab8380-0521-11ea-9c4b-8637e8c5f609.png)

Reply:
Hi,

I add a new option, named "Other", that option opens a field above, that you can put in blank or specify you configuration.

For example:

```
params = {
    "aws_access_key_id": key,
    "aws_secret_access_key": secret,
    "region_name": 'fr-par',
    "endpoint_url": 'http://s3.fr-par.scw.cloud',
}

resource = boto3.resource(
    's3',
    **params,
)

for bucket in resource.buckets.all():
    print(bucket.name)
```
![image](https://user-images.githubusercontent.com/1975978/68670200-1b6bbe80-052b-11ea-9dcb-d6edc9df28f0.png)

Reply:
yes please, rollback to selection